pyo3-derive-backend/src/method.rs

```diff
@@ -291,7 +290,10 @@ fn parse_attributes(attrs: &mut Vec<syn::Attribute>) -> syn::Result<(FnType, Vec
                 if name.is_ident("new") || name.is_ident("__new__") {
                     res = Some(FnType::FnNew)
                 } else if name.is_ident("init") || name.is_ident("__init__") {
-                    res = Some(FnType::FnInit)
+                    return Err(syn::Error::new_spanned(
+                        name,
+                        "#[init] is duplicated from PyO3 0.9.0!",
```

Comment:
Should this be `#[init] is removed from PyO3 0.9.0!`? Since `#[init]` was never working I think we don't have to return an error.